demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_modal.scss

```diff
@@ -9,21 +9,61 @@
 
 //  object: o-modal
 
-//  object container for modal
+//  Keyframe animations for modal
+@keyframes modal-open {
+    from {
+        opacity: 0;
+        transform: scale(.6) translateY(10%);
+    }
+
+    to {
+        opacity: 1;
+        transform: scale(1) translateY(0);
+    }
+}
+
+@keyframes modal-close {
```

Comment:
just a small nitpick: when the user clicks inside the modal, above the title, the modal closes unexpectedly. This seems to happen because margin is used instead of padding, so that area is interpreted as being outside the dialog